lib/openshift/cluster_version.rb

```diff
@@ -1,19 +1,70 @@
 module BushSlicer
   class ClusterVersion < ClusterResource
     RESOURCE = "clusterversions"
-    def channel(user: nil, cached: true, quiet: false)
+    def channel(user: nil, cached: false, quiet: false)
```

Comment:
I prefer to keep `cached: true` as a convention everywhere and use `cached: false` calls when needed.

Reply:
done